distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso

```diff
@@ -31,69 +32,76 @@ polyglot java import org.enso.table.aggregations.StandardDeviation as StandardDe
 polyglot java import org.enso.table.aggregations.Sum as SumAggregator
 
 ## Result type for aggregate_columns validation
-   - key_columns: Vector of Columns from the table to group by
-   - valid_columns: Table structure to build as pairs of unique column name and Aggregate_Column
-   - problems: Set of any problems when validating the input
+   - key_columns: Vector of Columns from the table to group by.
+   - valid_columns: Table structure to build as pairs of unique column name and Aggregate_Column.
+   - problems: Set of any problems when validating the input.
+   - old_style: Boolean indicating if the input was in the old style.
 type Validated_Aggregate_Columns
     ## PRIVATE
-    Value (key_columns:(Vector Column)) (valid_columns:(Vector (Pair Text Aggregate_Column))) (problems:(Vector Any))
+    Value (key_columns:(Vector Column)) (valid_columns:(Vector (Pair Text Aggregate_Column))) (problems:(Vector Any)) (old_style:Boolean=False)
```

Comment:
```suggestion
    Value (key_columns:(Vector Column)) (valid_columns:(Vector (Pair Text Aggregate_Column))) (problems:(Vector Any)) (old_style:Boolean)
```

why have the default value here? all paths should just always set it, defaulting it allows us to forget to do so.